### DISCUSSION TITLE
Seemingly incorrect permissions message

### DISCUSSION THREAD

**Discussion by sophware** [Micke-K/IntuneManagement#40] [Q&A]
![image](https://user-images.githubusercontent.com/41669/153767103-4d7eeb14-e81c-480e-b786-9293cc525d31.png)

In a few areas (Azure Branding, Conditional Access, and Named Locations), this wonderful app is telling me I don't have access. Using the same account, I certainly do seem to have access. I just logged in and modified a CA policy via browser, for example.

**Comment by sophware** [comment]
oh, and here's the only red text in the console output:

![image](https://user-images.githubusercontent.com/41669/153767226-56c30b8d-1889-4633-ae59-9c810ef06447.png)

**Reply by Micke-K** [reply]
Hello!

There is s difference between your permissions and app permissions. 

The **Use Default Permissions** setting is enabled. If you disable this, the app will then detect that it is missing some of the permissions and a consent prompt is displayed.

In some environments, you might have access but not permissions to add rights for the app. That is why the setting is on and you can then turn it off globally or for each tenant. 

Cheers!
